packages/contract-addresses/addresses.json

```diff
@@ -125,6 +125,48 @@
             "positiveSlippageFeeTransformer": "0x0000000000000000000000000000000000000000"
         }
     },
+    "5": {
```

Comment:
looks like the rest of the file uses lowercase. let's keep them consistent! (downstream packages might be assuming that they are in lowercase...)